celery/worker/strategy.py

```diff
@@ -148,17 +148,7 @@ def task_message_handler(message, body, ack, reject, callbacks,
             eventer=eventer, task=task, connection_errors=connection_errors,
             body=body, headers=headers, decoded=decoded, utc=utc,
         )
-        if _does_info:
```

Comment:
Just for the sake of being “the first iteration of messing up the code”, try to get all of this code back here, except the `info` call itself.